src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getDetailsOfParents } from '../../helpers';
+
+export const PeoplePage: FC = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
```

Comment:
There is a typo in the prop `peoples`. It should be `people` as it is more grammatically correct.